docs/aci-compose-features.md

```diff
@@ -0,0 +1,118 @@
+# Compose - Azure Container Instances mapping
+
+This document outlines mapping between `docker-compose.yaml` files and corresponding ACI equivalents.
+At a high-level, each Compose deployment is mapped to a single ACI container group. 
+Each service is mapped to a container in the container group, the Docker ACI integration provides no scaling of services. 
+
+## Compose fields mapping
+
+Table bellow list supported Compose file fields with equivalent ACI when supported.
+
+__Glossary:__
+
+- __✓:__ Converts
+- __n:__ Not yet implemented
+- __x:__ Not applicable / no 1-1 conversion
+
+| Keys                           | Map|  Notes                                                       |
+|--------------------------------|----|--------------------------------------------------------------|
+| __Service__                    | ✓  |                                                              |
+| service.service.build          | x  |  Ignored. no image build support on ACI.                              |
+| service.cap_add, cap_drop      | x  |                                                              |
+| service.command                | ✓  |  Override container Command. on ACI, specifying command will currently also override image entrypoint if the image has an Entrypoint defined |
+| service.configs                | n  |                                                              |
```

Comment:
Should this be 'x'? AFAIK ACI does not support the config concept?

Reply:
updated